nibabel/tests/test_helpers.py

```diff
@@ -1,15 +1,14 @@
 """ Helper functions for tests
 """
+from io import BytesIO
 
 import numpy as np
+from nose.tools import assert_true
```

Comment:
OK, glad to follow. Just to note, I was trying to follow the PEP8 convention for grouping imports into three: https://www.python.org/dev/peps/pep-0008/#imports . I'm not arguing for following that convention here, just to share where I was coming from.